platform/linux-generic/_fdserver.c

```diff
@@ -4,6 +4,10 @@
  * SPDX-License-Identifier:     BSD-3-Clause
  */
 
+#ifdef HAVE_CONFIG_H
```

Comment:
@muvarov. Ghm. How would source file check if config.h is available then? The usual story is that `AC_CONFIG_HEADER`/`AM_CONFIG_HEADER` would define `HAVE_CONFIG_H` and then application source file should include that file. If `HAVE_CONFIG_H` is not defined, `config.h` is nout available and thus application should not even try including it.